src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+div.red {
+  height: 100px;
```

Comment:
you duplicate this rule, try to figure out how to write it only once

Reply:
you fix Ostap comment, but div(same as other tags) is too general for styling, will be better to add common class